src/material/menu/menu-trigger.ts

```diff
@@ -291,6 +291,15 @@ export class MatMenuTrigger implements AfterContentInit, OnDestroy {
     }
   }
 
+  /**
+   * Updates the position of the menu to ensure that it fits all options within the viewport.
+   */
+  updatePosition(): void {
+    if (this._overlayRef) {
+      this._overlayRef.updatePosition();
```

Comment:
nit: can be reduce to `this._overlayRef?.updatePosition()`.

Reply:
:+1: wasn't sure which was preferred here.